docs/getting-started.md

```diff
@@ -328,6 +328,145 @@ mosquitto_sub -u <thing_id> -P <thing_key> -t channels/<channel_id>/messages -h
 If you are using TLS to secure MQTT connection, add `--cafile docker/ssl/certs/ca.crt`
 to every command.
 
+## CLI
+
+Mainflux CLI is another way to manage users, things, channels and messages.
+
+CLI can be downloaded as separate asset from [project realeses](https://github.com/mainflux/mainflux/releases) or it can be built with:
+
+```
+make cli
+```
+
+which will build `mainflux-cli` in `<project_root>/build` folder.
+
+Executing `build/mainflux-cli` without any arguments will output help with all available commands and flags:
+
+```
+Usage:
+  mainflux-cli [command]
+
+Available Commands:
+  channels    Manipulation with channels
+  help        Help about any command
+  msg         Send or retrieve messages
+  things      things <options>
+  users       users create/token <email> <password>
+  version     Get version of Mainflux Things Service
+
+Flags:
+  -c, --content-type string    Mainflux message content type (default "application/senml+json")
+  -h, --help                   help for mainflux-cli
+  -a, --http-prefix string     Mainflux http adapter prefix (default "http")
+  -i, --insecure               Do not check for TLS cert
+  -l, --limit uint             limit query parameter (default 100)
+  -m, --mainflux-url string    Mainflux host URL (default "http://localhost")
+  -o, --offset uint            offset query parameter
+  -t, --things-prefix string   Mainflux things service prefix
+  -u, --users-prefix string    Mainflux users service prefix
+
+Use "mainflux-cli [command] --help" for more information about a command.
+```
+
```

Comment:
Just add an info that you can have a help for each command - for example `mainflux-cli channels -h`.